scripts/pipeline/create_all_mlss.pl

```diff
@@ -424,7 +424,7 @@ sub make_named_species_set_from_XML_node {
     }
     my $method = $compara_dba->get_MethodAdaptor->fetch_by_type($xml_msa->getAttribute('method'));
     my $species_set = make_named_species_set_from_XML_node($xml_msa, $method, $division_genome_dbs);
-    push @mlsss, @{ Bio::EnsEMBL::Compara::Utils::MasterDatabase::create_multiple_wga_mlsss($compara_dba, $method, $species_set, ($xml_msa->getAttribute('gerp') // 0), $no_release) };
+    push @mlsss, @{ Bio::EnsEMBL::Compara::Utils::MasterDatabase::create_multiple_wga_mlsss($compara_dba, $method, $species_set, ($xml_msa->getAttribute('gerp') // 0), $no_release, undef, ($xml_msa->getAttribute('url') // 0), $no_release) };
```

Comment:
I'd suggest to remove the default value for `$xml_msa->getAttribute('url')`, because setting the default URL to `0` risks setting this value as the URL of WGA MLSSes created without one. Also, I would drop the second instance of `$no_release` as it is not needed.
```suggestion
    push @mlsss, @{ Bio::EnsEMBL::Compara::Utils::MasterDatabase::create_multiple_wga_mlsss($compara_dba, $method, $species_set, ($xml_msa->getAttribute('gerp') // 0), $no_release, undef, $xml_msa->getAttribute('url')) };
```